### PR TITLE
Fix docs styles

### PR DESCRIPTION
**Pull request by yuler**
- [x] Fix linting errors
- [x] Label has been added

## Change information

### CODE REVIEW

File: src/components/examples/footer.vue [hunk 1]
```diff
@@ -25,13 +25,19 @@ export default {
 </style>
 
 <style lang="stylus">
++layout(xs)
+  .ex-attributes .zi-card
+    display flex
+    overflow scroll
+    &::after
+      content ''
+      padding-right var(--geist-gap)
```
Comment (unix) [inline src/components/examples/footer.vue]: A blank line is required between two **style code blocks**.
```suggestion
      padding-right var(--geist-gap)
      
```

File: src/components/examples/footer.vue [hunk 2]
```diff
@@ -25,13 +25,19 @@ export default {
...
 .ex-footer-link
   margin-top rem(10)
   float right
   color var(--accents-7)
   font-size .8rem
   text-transform uppercase
-
```
Comment (unix) [inline src/components/examples/footer.vue]: require a blank line

File: src/components/examples/footer.vue [hunk 3]
```diff
@@ -25,13 +25,19 @@ export default {
 </style>
 
 <style lang="stylus">
++layout(xs)
+  .ex-attributes .zi-card
+    display flex
```
Comment (unix) [inline src/components/examples/footer.vue]: There may be side effects for multiple table examples.

demo (1)

File: src/pages/documents/document-slot.vue [hunk 4]
```diff
@@ -45,4 +45,5 @@
   display flex
   flex 1
   flex-direction column
+  overflow hidden
```
Comment (unix) [inline src/pages/documents/document-slot.vue]: Is this repairing something? It also seems to have side effects on `zi-popover`.

**Comment by yuler** [comment]
@unix 

Thank you for your advice. I've improved

**Pull request merged**